plugins/base/src/test/kotlin/content/annotations/SinceKotlinTest.kt

```diff
@@ -18,8 +28,208 @@ class SinceKotlinTest : BaseAbstractTest() {
+                        Platform.native to Version("1.3")
+                    )
+
+                    for(i in sinceKotlin) {
+                        val tag =
+                            find { it.sourceSets.first().analysisPlatform == i.key }?.documentation?.values?.first()
+                                ?.dfs { it is CustomTagWrapper && it.name == "Since Kotlin" }
+                                .assertNotNull("SinceKotlin[${i.key}]")
+                        assertEquals((tag.children.first() as Text).body, i.value.toString())
+                    }
+                }
+            }
+        }
+    }
+
+    @Test
+    fun `mpp fun with SinceKotlin annotation`() {
+        val configuration =   dokkaConfiguration {
+            sourceSets {
+                sourceSet {
+                    sourceRoots = listOf("src/")
+                    analysisPlatform = "jvm"
+                }
+                sourceSet {
+                    sourceRoots = listOf("src/")
+                    analysisPlatform = "native"
+                }
+                sourceSet {
+                    sourceRoots = listOf("src/")
+                    analysisPlatform = "common"
+                }
+                sourceSet {
+                    sourceRoots = listOf("src/")
+                    analysisPlatform = "js"
+                }
+            }
+        }
+        testInline(
+            """
+            |/src/main/kotlin/test/source.kt
+            |package test
+            |
+            |/** dssdd */
+            |@SinceKotlin("1.3")
+            |fun ring(abc: String): String {
+            |    return "My precious " + abc
+            |}
+        """.trimIndent(), configuration
+        ) {
+            documentablesTransformationStage = { module ->
+                @Suppress("UNCHECKED_CAST") val funcs = module.children.single { it.name == "test" }
+                    .children.filter { it.name == "ring" && it is DFunction } as List<DFunction>
+                with(funcs) {
+                    val sinceKotlin = mapOf(
+                        Platform.common to Version("1.3"),
+                        Platform.jvm to Version("1.3"),
+                        Platform.js to Version("1.3"),
+                        Platform.native to Version("1.3")
+                    )
+
+                    for(i in sinceKotlin) {
+                        val tag =
+                            find { it.sourceSets.first().analysisPlatform == i.key }?.documentation?.values?.first()
+                                ?.dfs { it is CustomTagWrapper && it.name == "Since Kotlin" }
+                                .assertNotNull("SinceKotlin[${i.key}]")
+                        assertEquals((tag.children.first() as Text).body, i.value.toString())
+                    }
+                }
+            }
+        }
+    }
+
     @Test
     fun `function with since kotlin annotation`() {
+        System.setProperty("dokka.SinceKotlin", "")
```

Comment:
Can't tell if it's intentional or a leftover - does this test check that there's no `SinceKotlin` handling if the property is not set? If so, can we reflect that in the name of the test?